packages/dashboard/src/contexts/DashContext/DashContext.ts

```diff
@@ -13,6 +13,7 @@ export interface ContextValue {
       lifecycle: ReceivedMessageLifecycle<DashboardProviderMessage>
     ) => any;
     toggleNotice: () => void;
+    updateAnalyticsConfig: (value: boolean) => void;
```

Comment:
So for this sorta thing, I'll also note that, in situations like this, you might consider getting into the habit of making separate operations for enable vs. disable. It feels a bit more hygienic: `enableAnalytics()` and `disableAnalytics()` are more clear and succinct compared to `updateAnalyticsConfig(true)` and `updateAnalyticsConfig(false)`.

(There are downsides here, of course, like having to pass an extra function around in the context here, but my personal stance is that it's worth it to have operations align with specific user actions)